components/input/apiExamples/disabled.html

```diff
@@ -0,0 +1,6 @@
+<auro-input disabled type="month-day-year" required>
+  <slot slot="label">Departure date</slot>
+</auro-input>
+<auro-input disabled bordered type="month-day-year">
+  <slot slot="label">Arrival date</slot>
+</auro-input>
```

Comment:
Add empty line at end of file.

Reply:
`input` will ultimately be replaced once we switch to the new `form` component, and deprecate the current components.

I have tracked this issue in a new ticket: https://github.com/AlaskaAirlines/auro-input/issues/352